property-services/src/main/java/org/egov/pt/consumer/FileStoreConsumer.java

```diff
@@ -97,7 +97,7 @@ private Property searchProperty(String id, String tenantId){
 
         PropertyCriteria criteria = PropertyCriteria.builder().tenantId(tenantId).uuids(Collections.singleton(id)).build();
 
-        List<Property> properties = propertyRepository.getProperties(criteria, false);
```

Comment:
Why is this needed?

Reply:
In this getProperties() method three parameter need to pass but only two are passing
that's why added third parameter in this method call